pkgs/by-name/jo/jo/package.nix

```diff
@@ -5,33 +5,48 @@
   autoreconfHook,
   pandoc,
   pkg-config,
+  versionCheckHook,
+  nix-update-script,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "jo";
   version = "1.9";
 
   src = fetchFromGitHub {
     owner = "jpmens";
     repo = "jo";
-    rev = version;
+    tag = finalAttrs.version;
     sha256 = "sha256-1q4/RpxfoAdtY3m8bBuj7bhD17V+4dYo3Vb8zMbI1YU=";
   };
 
-  enableParallelBuilding = true;
```

Comment:
Is there a particular reason for this?

Reply:
the default value is true, so we needn't add this.

Reply:
Isn't this false by default?

https://nixos.org/manual/nixpkgs/unstable/#sec-config-options-reference